<launch>
  
  <!-- view generator -->
  <!-- <include file="$(find scitos_moveit_config)/launch/move_group.launch"/> -->
  <node name="ptu_fk" pkg="viper" type="ptu_fk"/>

  <rosparam param="roi">[[2.0,-8.0],[5.0,-8.0],[5.0,2.0],[2.0,2.0]]</rosparam>   
  <param name="inflation_radius" type="double" value="0.7" />  
  <param name="views_at_pose" type="int" value="8" />  

  <arg name="map" default="/map"/>  
  <include file="$(find nav_goals_generator)/launch/nav_goals_generator.launch">
    <arg name="map" value="$(arg map)"/>
  </include>


  <!-- view evaluation --> 
  <param name="frustum_near" type="double" value="0.8"/>
  <param name="frustum_far" type="double" value="2.5"/>
  <param name="frustum_angle" type="double" value="40.5"/> <!-- vertical FOV (focal length 32.5) -->

  
  <!-- plan execution --> 
	
  <!--<include file="$(find strands_morse)/bham/launch/bham_cs_nav2d.launch"/>-->
  <!-- <include file="$(find scitos_ptu)/launch/ptu_action_server.launch"/> -->
  
</launch>